.github/CONTRIBUTING.md

```diff
@@ -1,83 +1,64 @@
 # Contributing
-
-Before making any contributions, please sign one of the contributor
-license agreements below.
-
-Fork the repo, develop and test your code changes.
+Before making any contributions, please sign one of the contributor license agreements below. Fork the repo, develop and test your code changes.
 
 Install all dependencies including development requirements by running:
 
 ``` sh
-$ npm install -d
+$ npm install
 ```
 
-Tests are run using mocha. To run all tests just run:
+Builds or tests require the installation of [`make`](https://www.gnu.org/software/make/).  To perform the first build, go ahead and run it!
 
 ``` sh
+$ make
+```
+
+Next, go ahead and run the tests:
+
+```sh
 $ npm test
 ```
 
-which looks for tests in the `test/` directory.
+This looks for tests in the `test/` directory.  In addition to running the tests, this will also check code formatting.  Code formatting (and other issues) can be fixed automatically by running:
 
-Your code should honor the [Google JavaScript Style Guide][js-guide].
-You can use [Closure Linter][c-linter] to detect style issues.
+```sh
+$ make fix
+```
 
-Submit a pull request. The repo owner will review your request. If it is
-approved, the change will be merged. If it needs additional work, the repo
-owner will respond with useful comments.
+Submit a pull request. The repo owner will review your request. If it is approved, the change will be merged. If it needs additional work, the repo owner will respond with useful comments.
```

Comment:
Suggestions: 

The repo owner reviews your request. They can approve and merge your change. If your change needs additional work, they follow up with comments.